src/style.css

```
@@ -0,0 +1,148 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  max-width: 200px;
```

```suggestion
  width: 200px;
```
You can make it just a width